replibyte/src/main.rs

```diff
@@ -290,6 +292,15 @@ fn main() -> anyhow::Result<()> {
                     },
                 };
 
+                if cmd.local {
+                    // TODO: handle Postgres and MongoDB
+                    let task = FullRestoreTask::new(PostgresDocker::default(), bridge, options);
+                    let _ = task.run(progress_callback)?;
+
+                    // TODO: Return the credentials to the user.
```

Comment:
We'll also need to keep the container running until the user press CTRL/CMD + C (Sigterm) that we need to handle to drop the container.

Reply:
As a default, the container is stopped when the user press `ctrl+c` and `--remove` option was added to definitely remove the container.